app/views/service_providers/_form.html.erb

```diff
@@ -75,7 +75,7 @@
                    input_html: { class: 'usa-input' } %>
   <% end %>
 
-  <% if Figaro.env.logo_upload_enabled == 'true' %>
+  <% if ENV.fetch('logo_upload_enabled', 'true') == 'true' %>
```

Comment:
What's with the switch from Figaro to raw environment variables?

Don't we use Figaro populate environment variables from the YML files? Like my expectation is that if we left it as Figaro, it would accept env vars too, so that's easier to work with?